basis/english/english.factor

```diff
@@ -141,11 +143,80 @@ PRIVATE>
         [ "s" append ]
     } cond match-case ;
 
+: singular? ( word -- ? )
+    [ singularize ] [ = ] bi ;
+
+: plural? ( word -- ? )
+    >lower {
+        { [ dup empty? ] [ drop f ] }
+        { [ dup plural-to-singular key? ] [ drop t ] }
+        { [ dup dup plural-to-singular at = ] [ drop t ] } ! moose -> moose
```

Comment:
Does this case run?  Wouldn't the ``[ dup plural-to-singular key? ]`` before it return ``t``?

Reply:
Maybe you mean to use ``singular-to-plural`` there.

Reply:
I think it was a guard for an earlier iteration because it kept failing to say that `moose` etc are both singular and plural, but you're right it's redundant now